PyInstaller/building/api.py

```diff
@@ -154,7 +154,7 @@ class PKG(Target):
                  'DEPENDENCY': 'd'}
 
     def __init__(self, toc, name=None, cdict=None, exclude_binaries=0,
-                 strip_binaries=False, upx_binaries=False):
+                 strip_binaries=False, upx_binaries=False, exclude_upx=[]):
```

Comment:
Using a list as a default parameter value is bad practice in python: if one changes this list, this will effect the default parameter value, too (since lists are passed by reference) - which is a common pitfall. So this should be `exclude_upx=None` ...